library/tinyspline.c

```diff
@@ -467,8 +467,8 @@ void ts_internal_bspline_evaluate(
                             a * deBoorNet->points[ridx++];
                 }
             }
-            lidx += dim;
-            ridx += dim;
+            lidx += (int)dim;
```

Comment:
I think using `size_t` rather than `int` for `lidx` and `ridx` is more reasonable to fix the error. They should never be negative anyway (as far as I can see). What do you think @giumas?